src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -169,12 +211,68 @@ export const ObservationCreate = ({
     value,
   ]);
 
+  const checkAccuracyAndLocation = React.useCallback(() => {
+    const confirmationOptions: AlertButton[] = [
+      {
+        text: formatMessage(m.saveAnyway),
+        onPress: createObservation,
+        style: 'default',
+      },
+      {
+        text: formatMessage(m.manualEntry),
+        onPress: () => navigation.navigate('ManualGpsScreen'),
+        style: 'cancel',
+      },
+      {
+        text: formatMessage(m.keepWaiting),
+        onPress: () => {},
+      },
+    ];
+
+    if (!value) {
+      noop();
+      return;
+    }
+
+    // If the user has already inputted a manual location, do not check if location is accurate
+    if (value.metadata?.manualLocation) {
+      createObservation();
+    }
+
+    const accuracy = value.metadata?.position?.coords?.accuracy;
+
+    if (!liveLocation) {
+      Alert.alert(
+        formatMessage(m.noGpsTitle),
+        formatMessage(m.noGpsDesc),
+        confirmationOptions,
+      );
+      return;
+    }
+
+    // If we don't have accuracy, allow save anyway (this is a remnant from mapeo: https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/SaveButton.js#L125C3-L125C50)
+    if (
+      accuracy &&
+      typeof accuracy === 'number' &&
+      accuracy < MINIMUM_ACCURACY
```

Comment:
This should be checking if the accuracy is too large (since accuracy is the numeric range of distance in this case)

```suggestion
      accuracy >= MINIMUM_ACCURACY
```

Reply:
Im a little confused here. I think you are right, but historically we have used `accuracy < MINIMUM_ACCURACY` in comapeo mobile. Why has it worked before?

Reply:
see: https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/SaveButton.js#L126

Reply:
Mapeo Mobile performs a different check. It's checking to see that the accuracy is valid and good enough (i.e. `accuracy < 10`) first and if it is, goes straight ahead to saving the draft.

https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/SaveButton.js#L85-L95

https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/SaveButton.js#L116-L127

Your PR is doing checking for cases where it's invalid or insufficient first, in order to potentially open the dialogs about manually setting a coordinate.

Reply:
RIght....I should probably just read the code 😬

Reply:
Yeah i think the `MINIMUM_ACCURACY` naming is a bit confusing since accuracy is one of those inverted quantities (in this case at least). Ideally it would be more descriptive in this context 😅

Reply:
https://github.com/digidem/comapeo-mobile/pull/702/commits/9ed6b36b9c35fd46ccb358820f0d0980cfc3aab7